Blockzilla/ar.lproj/Localizable.strings

```diff
@@ -2,7 +2,7 @@
 "About.learnMoreButton" = "اطّلع على المزيد";
 
 /* Label on About screen */
-"About.missionLabel" = "\u200f%@ منتج من موزيلا. مهمتنا تشجيع إنترنت مفتوح و صحي.";
```

Comment:
I don't understand what happened here.

Reply:
Oh! When I do a diff on the command line, I actually see this:

```
 /* Error label when the blocker is not enabled, shown in the intro and main app when disabled */
-"Safari.instructionsNotEnabled" = "\u200f%@ ليس مفعّلا.";
+"Safari.instructionsNotEnabled" = "<U+200F>%@ ليس مفعّلا.";
```

Which does look good?

Reply:
@flodolo do you have an opinion about escaping `\u200f` vs directly putting the raw code `<U+200F>` in the string?

Reply:
I prefer the escaping, as the raw code is not used anywhere in our products.

Reply:
Ok so I will figure out a way to make that work.

The way GitHub visualizes these diffs is a bit of a mystery .. I think it is interpreting the raw UTF-8, which renders correctly then, but is not the rigyt thing to do in a diff I think. Maybe I will file a bug for that.